src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -262,20 +262,17 @@ export function TitleField({style={}, value, label='Title:', size=30}) {
     );
 }
 
-export function ZipStructure({fieldKey='zipType'}) {
+export function FileStructure({fieldKey='zipType'}) {
     return (
-        <ListBoxInputField
-            fieldKey = {fieldKey}
-            initialState = {{
-                tooltip: 'File Structure',
-                label : 'File Structure:'
-            }}
-            options = {[
-                {label: 'Structured (with folders)', value: 'folder'},
-                {label: 'Flattened (no folders)', value: 'flat'}
-            ]}
-        />
-
+        <Stack width={250} mt={2}>
+            <SwitchInputField
+                fieldKey={fieldKey}
+                initialState={{ value: 'folder' }}
+                trueValue='flat'
+                falseValue='folder'
+                label='Flattened File Structure'
+            />
+        </Stack>
```

Comment:
fieldKey can be defined something as `isFlattenedStructure` and the payload will have true or false. 

Then consumer of that payload (I'm assuming your server-side code) can read true as flat and false as folder.